web/src/app/schedules/temp-sched/TempSchedDialog.tsx

```diff
@@ -72,33 +72,30 @@ export default function TempSchedDialog({
     key: number
   }
   function renderSlide({ index, key }: SlideRenderer): ReactNode {
-    switch (index) {
-      case 0:
-        return (
-          <TempSchedTimesStep
-            key={key}
-            stepText='STEP 1 OF 2'
-            scheduleID={scheduleID}
-            value={value}
-            edit={edit}
-          />
-        )
-      case 1:
-        return (
-          <TempSchedAddShiftsStep
-            key={key}
-            value={value.shifts}
-            onChange={(shifts: Shift[]) => setValue({ ...value, shifts })}
-            stepText={edit ? '' : 'STEP 2 OF 2'}
-            scheduleID={scheduleID}
-            start={value.start}
-            end={value.end}
-            edit={edit}
-          />
-        )
-      default:
-        return null
+    if (index === 1) {
+      return (
+        <TempSchedAddShiftsStep
+          key={key}
+          value={value.shifts}
+          onChange={(shifts: Shift[]) => setValue({ ...value, shifts })}
+          stepText={edit ? '' : 'STEP 2 OF 2'}
+          scheduleID={scheduleID}
+          start={value.start}
+          end={value.end}
+          edit={edit}
+        />
+      )
     }
+
+    return (
+      <TempSchedTimesStep
+        key={key}
+        stepText='STEP 1 OF 2'
+        scheduleID={scheduleID}
+        value={value}
+        edit={edit}
+      />
+    )
```

Comment:
Looks like we should be comparing the indexes for all cases instead of having a default return here. The field validation for the time step no longer shows the error messages but does still disable the next button on error.

Reply:
I am seeing field validation coming through on my end- was there a specific set of actions you did to reproduce?

Reply:
Looks like this is a chrome issue. So to recreate on chome, setting the end time before the start time disables the next button but shows no field validation.

Reply:
Updating my yarn to 1.22.10 seems to resolve this issue.